scripts/histmakers/mw_with_mu_eta_pt.py

```diff
@@ -51,9 +53,22 @@
 wprocs = ["WplusmunuPostVFP", "WminusmunuPostVFP", "WminustaunuPostVFP", "WplustaunuPostVFP"]
 zprocs = ["ZmumuPostVFP", "ZtautauPostVFP"]
 
+# custom template binning
+# round as precaution because e.g. sometimes 2.4 gets read as 2.3999 in python (1 digit would be enough but just in case)
+tokens_eta = args.eta.split(',')
+template_neta = int(tokens_eta[0])
+template_mineta = round(float(tokens_eta[1]), 2)
```

Comment:
What's the purpose of rounding here?

Reply:
Well, I thought the comment was saying that :p 
The fact is that sometimes in past scripts the number was not rounded to the value one expects, and this was causing issues when used to define histogram binning.
It was actually more frequent when creating the whole array of bin edges (rather than just the two extremes as here), where some edges were not rounded correctly and then causing mismatches when trying to compare histograms from different sources.

Reply:
Ah, I see, this is because the argument is provided as a string.  It should be possible to directly provide this as an array of floats instead, e.g. with nargs=6, type=float for the parser.

Reply:
Yeah, it is a possibility, passing strings seemed more compact but I can change it. The number of bins needs to be converted to int anyway, though

Reply:
Especially if having these as floats avoids this numerical imprecision + need for rounding I think this is better.

Reply:
Done, now it works as --eta 48 -2.4 2.4  --pt 29 26 55 (default values are already those we use normally)

Reply:
I'll try to resist my urge to be nitpick, but do we really need this? If it's purely for the eta < 1.0 study this could be a single boolean flag...

Reply:
Possibly also with a ptmin argument. But I don't think we ever considered changing from 1 GeV pt bins (would be useful to check the variable binning, though)